scripts/download.js

```diff
@@ -0,0 +1,237 @@
+    console.log("usedHnsLink =  " + usedHnsLink);
+  } else {
+    usedSkylink = process.argv[2];
+    console.log("\n\n\nusedSkylink =  " + usedSkylink);
+    usedHnsLink = process.argv[3];
+    console.log("usedHnsLink =  " + usedHnsLink);
+  }
+
+  //1. use downloadFile to get a file from skynet.
+  async function downloadFile(path, skylink) {
+    await client
+      .downloadFile(path, skylink)
+      .then((res) => {
+        console.log("\n\n\n1. use downloadFile to get a file from skynet.");
+        console.log("skylinkUrl: " + res);
+      })
+      .catch((err) => {
+        console.log("\n1. Get Error: ", err);
+      });
+  }
+
+  //2. use downloadFileHns to get a hns file from skynet.
+  async function downloadFileHns(path, hnsLink) {
+    await client
+      .downloadFileHns(path, hnsLink)
+      .then((res) => {
+        console.log("\n\n\n2. use downloadFileHns to get a hns file from skynet.");
+        console.log("hnsLink: " + res);
+      })
+      .catch((err) => {
+        console.log("\n2. Get Error: ", err);
+      });
+  }
+
+  //3. use getSkylinkUrl to get the url from a skylink.
+  async function getSkylinkUrl(skylink) {
+    await client
+      .getSkylinkUrl(skylink)
+      .then((res) => {
+        console.log("\n\n\n3. use getSkylinkUrl to get the url from a skylink.");
+        console.log("skylinkUrl: " + res);
+      })
+      .catch((err) => {
+        console.log("\n3. Get Error: ", err);
+      });
+  }
+
+  //4. use getHnsUrl to get the hnsUrl from a hnsLink.
+  async function getHnsUrl(hnsLink) {
+    await client
+      .getHnsUrl(hnsLink)
+      .then((res) => {
+        console.log("\n\n\n4. use getHnsUrl to get the hnsUrl from a hnsLink.");
+        console.log("hnsUrl: " + res);
+      })
+      .catch((err) => {
+        console.log("\n4. Get Error: ", err);
+      });
+  }
+
+  //5. use getHnsresUrl to get the hnsresUrl from a hnsLink.
+  async function getHnsresUrl(hnsLink) {
+    await client
+      .getHnsresUrl(hnsLink)
+      .then((res) => {
+        console.log("\n\n\n5. use getHnsresUrl to get the hnsresUrl from a hnsLink.");
+        console.log("hnsresUrl: " + res);
+      })
+      .catch((err) => {
+        console.log("\n5. Get Error: ", err);
+      });
+  }
+
+  //6. use getMetadata to get the metadata from a skylink.
+  async function getMetadata(skylink) {
+    await client
+      .getMetadata(skylink)
+      .then((res) => {
+        console.log("\n\n\n6. use getMetadata to get the metadata from a skylink.");
+        console.log("Metadata: " + JSON.stringify(res));
+      })
+      .catch((err) => {
+        console.log("\n6. Get Error: ", err);
+      });
+  }
+
+  //7. use getFileContent to get the fileContent from a skylink.
+  async function getFileContent(entryLink) {
+    await client
+      .getFileContent(entryLink)
+      .then((res) => {
+        console.log("\n\n\n7. use getFileContent to get the fileContent from a skylink.");
+        console.log("data: " + res.data);
```

Comment:
When I tried running this script, there was way too data at this step and it took forever to print out. Also, it was mostly gibberish. I would remove this line.

Reply:
hello m-cat,

I didn't know how to program it differently so I could show them that everything worked.

But I can exclude all fields with data, so that the console output is less. 
And just specify "done" in the console output.

Reply:
Sounds great!